projects/gstreamer/build.sh

```diff
@@ -21,12 +21,20 @@ PREFIX=$WORK/prefix
 PLUGIN_DIR=$PREFIX/lib/gstreamer-1.0
 export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig
 mkdir -p $PREFIX
+export PATH=$PREFIX/bin:$PATH
 cd $WORK
 
 # Minimize gst-debug level/code
 export CFLAGS="$CFLAGS -DGST_LEVEL_MAX=2"
+#export LD=$CXX
```

Comment:
I tried to remove this line via GitHub's web interface, and CLA started to complain, weird.

Could you please remove that and create a new PR, so it won't complain anymore?

Reply:
whoops, I'll remove it indeed

Reply:
Reuploaded with fixed commits. googlebot seems happy now :)